.github/workflows/build.yml

```diff
@@ -130,23 +130,26 @@ jobs:
       - run: openssl genrsa -out ./config/jwt.key 2048 && chmod 0777 ./config/jwt.key
       - run: npm run swagger:validate
       - run: npm run coverage-ci # Separate command to limit the number of workers to prevent timeouts
-      - name: "Post Migration test-suite"
-        if: ${{ matrix.typeorm-connection == 'mariadb' }}
-        run: npm run test-ci-migrate
       - run: git config --global --add safe.directory $GITHUB_WORKSPACE # To avoid dubious ownership
         if: ${{ matrix.typeorm-connection == 'mariadb' }}
 
       - name: "Cannot commit code coverage cross-fork"
         if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork && matrix.typeorm-connection == 'mariadb'
         run: |
             echo "::warning Cannot comment code coverage cross-fork"
+      - name: "Fetch target branch coverage from cache"
+        if: github.event_name == 'pull_request' && !github.event.pull_request.head.repo.fork && matrix.typeorm-connection == 'mariadb'
+        uses: actions/cache/restore@v3
+        with:
+          path: reports/coverage-base/
+          key: ${{ runner.os }}-coverage-${{ matrix.typeorm-connection }}-${{ github.base_ref }}
       - name: "Comment code coverage on PR"
         if: github.event_name == 'pull_request' && !github.event.pull_request.head.repo.fork && matrix.typeorm-connection == 'mariadb'
         uses: sidx1024/report-nyc-coverage-github-action@v1.2.6
```

Comment:
One other thing, `@v1.2.7` fixes the `set-output` deprecation messages.

Reply:
v1.2.7? of the action?

Reply:
Of `sidx1024/report-nyc-coverage-github-action` yes, I only selected one line when I added the comment but GH decided you needed more lines.